Nautilus/Patchers/OptionsPanelPatcher.cs

```diff
@@ -91,6 +92,7 @@ internal static void AddTabs_Postfix(uGUI_OptionsPanel __instance)
         // Maybe this could be split into its own file to handle nautilus options, or maybe it could be removed alltogether
         optionsPanel.AddHeading(modsTab, "Nautilus");
         optionsPanel.AddToggleOption(modsTab, "Enable debug logs", Utility.InternalLogger.EnableDebugging, Utility.InternalLogger.SetDebugging);
+        optionsPanel.AddToggleOption(modsTab, "Enable Mod Databank Entries", ModDatabankHandler.isEnabled);
```

Comment:
I noticed that this is the only Nautilus option that capitalizes every word as opposed to just the first (`Enable Mod Databank Entries` vs `Enable mod databank entries`)

Reply:
I'm not very good at naming conventions haha